modules/perception/camera/app/lane_camera_perception.cc

```diff
@@ -69,12 +69,13 @@ void LaneCameraPerception::InitLane(
     const std::string &work_root, base::BaseCameraModelPtr &model,
     const app::PerceptionParam &perception_param) {
   // Init lane
-  CHECK(perception_param.has_lane_param()) << "Failed to include lane_param";
-  {
+  CHECK(perception_param_.lane_param_size() > 0)
```

Comment:
`CHECK_GT(perception_param_.lane_param_size(), 0)`

Reply:
Thank you for the review! Fixed the code.